src/sentry/scripts/similarity/index.lua

```diff
@@ -570,9 +571,13 @@ local function search(configuration, parameters, limit)
         table.sort(
             candidates,
             function (this, that)
-                if this.score > that.score then
+                if this.avg_hits > that.avg_hits then
```

Comment:
This also now mirrors the sort implementation defined in the Python backend: https://github.com/getsentry/sentry/blob/5798b19a6172ce540d0677f4ad160441d2db90d7/src/sentry/similarity/backends/redis.py#L74-L78